devicemodel/hw/platform/vssram/vssram.c

```diff
@@ -458,10 +457,19 @@ static int vssram_ept_map_buffer(struct vmctx *ctx, struct vssram_buf *buf_desc)
 	memmap.vma_base = buf_desc->vma_base;
 	memmap.user_vm_pa = buf_desc->gpa_base;
 	memmap.len = buf_desc->size;
+
+	/*
+	 * This is a Walk-around:
+	 * - As only MMIO type can be supported when doing EPT unmap in HSM driver,
+	 *   so hardcoding of 'type' to be MMIO type.
+	 */
+	memmap.type = ACRN_MEMMAP_MMIO;
```

Comment:
What's the memory type of MMIO EPT page entry? If it is UC, then SSRAM can not work...

Reply:
@ywan170  This MMIO type is for Unmap case only and is a Walk Around solution to by-bass a check in ACRN kernel.
 fix for upstream is still in-progress.